apps/condo/domains/common/components/MenuItem.tsx

```diff
@@ -159,13 +111,30 @@ export const MenuItem: React.FC<IMenuItemProps> = (props) => {
 
     const menuItemIdProp = id ? { id: id } : {}
 
+    const baseClass = styles.menuItemWrapper
+    const justifyClass = isCollapsed ? styles.justifyCenter : ''
+    const activeClass = isActive ? styles.active : ''
+    const disabledClass = disabled ? styles.disabled : ''
+    const externalClass = wrapperClassName || ''
+
+    const menuItemClassName = [baseClass, justifyClass, activeClass, disabledClass, externalClass, 'side'].filter(Boolean).join(' ')
```

Comment:
Use classnames to generate final classname, search for references across ui package or just by import